### ISSUE TITLE
Is it possible to share the config.yml for models?

### ISSUE THREAD

**Issue by SevenLJY** [bennyguo/instant-nsr-pl#8]
Hi,

I'm wondering if the showing case for the chair is using the same configuration as the neus-blender.yml you provided in the repo, and if it is possible to share the training params for several scenes as a guidance of hyper-params fine-tuning. Thanks in advance!

Currently, the chair model I trained using the provided .yml cannot reproduce the mesh as good as the one in your show case. Here is my test result (256 resolution mesh).

<img width="256" alt="Screen Shot 2022-11-06 at 9 53 16 AM" src="https://user-images.githubusercontent.com/26187268/200186826-76cb53c8-8309-45a1-a7f3-a9cc8f897fed.png">

**Comment by bennyguo** [comment]
All the results are produced using the given config file. This is probably due to the randomness during training and maybe you should try different seeds to see if results differ :)

**Comment by SevenLJY** [comment]
It indeed produces a slightly different mesh by changing the seed, but it still easily gets holes, especially on the flat surface.
